internal/namespaces/instance/v1/command_server_create_test.go

```diff
@@ -0,0 +1,216 @@
+package instance
+
+import (
+	"testing"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+)
+
+func Test_CreateServer(t *testing.T) {
+
+	//t.Run("Simple", core.Test(&core.TestConfig{
+	//	Commands: GetCommands(),
+	//	BeforeFunc: func(ctx *core.BeforeFuncCtx) error {
+	//		ctx.Meta["Server"] = ctx.ExecuteCmd("scw instance server create image=ubuntu_bionic")
+	//		return nil
+	//	},
+	//	Cmd: "scw instance server create image=ubuntu_bionic",
+	//	AfterFunc: func(ctx *core.AfterFuncCtx) error {
+	//		ctx.ExecuteCmd("scw instance server delete server-id={{ .Server.id }}")
+	//		return nil
+	//	},
+	//	Check: core.TestCheckCombine(
+	//		core.TestCheckGolden(),
+	//	),
+	//}))
+}
+
+func Test_CreateServerErrors(t *testing.T) {
+
+	////
+	// Image errors
+	////
+	t.Run("Error: missing image label", core.Test(&core.TestConfig{
+		Parallel: true,
+		Commands: GetCommands(),
+		Cmd:      "scw instance server create",
+		Check: core.TestCheckCombine(
+			core.TestCheckGolden(),
+			core.TestCheckExitCode(1),
+		),
+	}))
+
+	t.Run("Error: invalid image label", core.Test(&core.TestConfig{
+		Parallel: true,
+		Commands: GetCommands(),
+		Cmd:      "scw instance server create image=macos",
```

Comment:
maybe use an os name that doesn't exist at all

what if someday we support macos ?